code/weapon/weapons.cpp

```diff
@@ -225,6 +225,8 @@ const float HOMING_DEFAULT_FREE_FLIGHT_FACTOR = 0.25f;
 #define REARM_NUM_MISSILES_PER_BATCH 4              
 #define REARM_NUM_BALLISTIC_PRIMARIES_PER_BATCH 100 
 
+const float minimum_spawn_interval = 0.1f;
```

Comment:
Please use `static` on this, and also capitalize it. Makes it easier to recognize as a global when capitalized.